service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java

```diff
@@ -137,15 +140,11 @@ void shouldBuildOtherApplicationsBundleWithOtherApplicantName() {
         SupportingEvidenceBundle supportingDocument = createSupportingEvidenceBundle();
         PBAPayment pbaPayment = buildPBAPayment();
 
-        // select "Someone else"
-        DynamicList applicantsList = DynamicList.builder()
-            .value(DYNAMIC_LIST_ELEMENTS.get(1)).listItems(DYNAMIC_LIST_ELEMENTS).build();
-
         CaseData caseData = CaseData.builder()
             .additionalApplicationType(List.of(OTHER_ORDER))
             .temporaryOtherApplicationsBundle(createOtherApplicationsBundle(supplement, supportingDocument))
             .temporaryPbaPayment(pbaPayment)
-            .applicantsList(applicantsList)
+            .applicantsList(DYNAMIC_LIST_ELEMENTS.get(1).getCode()) // select "Someone else"
```

Comment:
Would it be better to just explicitly put `"Someone else"`

Reply:
yeah, updated